R/01-webdata-obj.R

```diff
@@ -103,6 +103,18 @@ setMethod("webdata", signature("character"), function(.Object=c("prism",  "iclus
   return(webdata)
 })
 
+#' @rdname webdata-methods
+#' @aliases webdata
+setMethod("webdata", signature("geojob"), function(.Object, ...) {
+  xmlDF <- readGeojobXML(.Object@xml)
```

Comment:
`xml(.Object)` wasn't working at one point, but maybe I just was sending the wrong class or something